lib/build.api.js

```diff
@@ -210,15 +235,15 @@ class Build {
           // Make sure the destination base path exists.
           if (!fs.existsSync(destBase)) {
             this.aquifer.console.log('Destination directory does not exist. Creating: ' + destBase, 'status');
-            fs.mkdirSync(destBase);
+            fs.mkdirsSync(destBase);
```

Comment:
Should probably leave a note that this is from `fs-extra`, https://github.com/jprichardson/node-fs-extra#mkdirsdir-callback